apply/default_applier.go

```diff
@@ -199,7 +199,10 @@ func (c *DefaultApplier) Apply() (err error) {
 		logger.Debug("sealer apply process %s", action)
 		err := ActionFuncMap[action](c)
 		if err != nil {
-			return err
+			if c.ClusterDesired.DeletionTimestamp == nil {
```

Comment:
The registry dir umount error we will fix it in runtime.reset. I think we should not ignore all the delete errors

Reply:
We already fixed this issue, I think you can just delete this

Reply:
ok i delete this